doc/api_base.rst

```diff
@@ -78,9 +78,9 @@
       base.repos.add(repo)
       base.fill_sack()
 
-  .. method:: do_transaction([display])
+  .. method:: do_transaction([displays])
```

Comment:
`do_transaction([display], [displays])` would be really ugly. Then the best option is probably to keep the `display` - make it accept TransactionDisplay or the list of TransactionDisplays and don't change API at all - just document the API addition. (Queries.filter(name=...) also takes one instance or the list). Agree?